libvips/foreign/webpsave.c

```diff
@@ -604,6 +615,10 @@ vips_foreign_save_webp_init_config(VipsForeignSaveWebp *webp)
 	webp->config.lossless = webp->lossless || webp->near_lossless;
 	webp->config.alpha_quality = webp->alpha_q;
 	webp->config.method = webp->effort;
+	webp->config.target_size = webp->target_size; 
+
+	if (webp->pass)
```

Comment:
You could also use `if (vips_object_argument_isset(object, "pass")) {}`, it's a little safer.

Reply:
Interesting. I'll read through the internals. This is my first contribution to this project, and I haven't seriously programmed in C for over a decade.

Reply:
I don't see a variable called `object` in scope here. Could I have a little more guidance?

Reply:
I think I figured out how to do it properly. Resolved.